### DISCUSSION TITLE
230705 팀문화(코드 리뷰) 회의 (14:00 ~ 14:50)

### DISCUSSION THREAD

**Discussion by Creative-Lee** [woowacourse-teams/2023-shook#18] [회의]
## 회의 안건
- 팀문화 - 코드 리뷰 형식에 대해 방향을 정한다.

**Comment by somsom13** [comment]
### 코드 리뷰 형식

- Merge 규칙
   - 각 파트 전원의 Approve를 받아야 한다.

- 코드 리뷰 시 커멘트 분류
   -  간단한 의견 및 질문: 문장의 앞에 💬 (speech balloon)을 붙인다. (ex. 💬 아코 이거 의견이 궁금해요~) 
   - 변경 제안 및 토론이 필요한 내용인 경우, 그냥 내용을 쓴다. (ex. 아코 filter를 사용하는건 어떨까요~?)
   - 변경 제안이 있는 경우: Request Change 선택 / 변경 제안이 없는 경우: Approve

- 리뷰 내용 규칙
   - https://tech.kakao.com/2022/03/17/2022-newkrew-onboarding-codereview/
   - https://soojin.ro/review/review-comments

- 리뷰에 대한 토론은 오프라인으로 대화한다.
   - 리뷰를 이해하지 못한 경우, 토론이 필요한 경우에는 대화를 한다.
   - 대화의 결론은 코멘트로 남겨둔다.

- 작업, 리뷰 반영 기간
   - 이슈 갯수에 상관 없이 작업 2 : 리뷰 및 반영 1 의 비율로 진행한다.
